packages/framework-provider-azure/src/library/read-model-adapter.ts

```diff
@@ -28,9 +100,9 @@ export async function storeReadModel(
   readModelName: string,
   readModel: ReadModelInterface
 ): Promise<void> {
-  await db
-    .database(config.resourceNames.applicationStack)
-    .container(config.resourceNames.forReadModel(readModelName))
-    .items.upsert(readModel)
-  logger.debug('[ReadModelAdapter#storeReadModel] Read model stored')
+  const version = readModel.boosterMetadata?.version || 0
```

Comment:
In TypeScript it's more common to use the `??` operator:

```suggestion
  const version = readModel.boosterMetadata?.version ?? 0
```

Reply:
Yep, always use `??` instead of `||` as the latter will accept falsy values, and `??` checks for nullables